INSTALL.md

```diff
@@ -1,3 +1,8 @@
 # Installation instructions
 
-Detailed instructions on how to install, configure, and get the project running.
+If you worked with a previous release of this package, first run:
+
+`rm -r node_modules/`
```

Comment:
I'd leave this out of the official docs and just put it in the PR message. In the future when more PRs are merged into the codebase it will become unclear whether this step is necessary or not.